client/src/lib.rs

```diff
@@ -1,4 +1,41 @@
-//! DAP protocol client
+//! A [DAP-PPM](https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/) client
```

Comment:
I would just say "DAP". PPM is the working group that is writing DAP, not the protocol itself.
```suggestion
//! A [DAP](https://datatracker.ietf.org/doc/draft-ietf-ppm-dap/) client
```